docs/HowToInstall.md

```diff
@@ -0,0 +1,77 @@
+# How to install?
+
+## Prerequisit
+
+ONE-vscode is an [extension](https://code.visualstudio.com/docs/editor/extension-marketplace) of [vscode](https://code.visualstudio.com/). To use ONE-vscode, you should install vscode first.
+
+### vscode and its extensions
+
+Let's install vscode according to [doc](https://code.visualstudio.com/docs/setup/setup-overview). (You should also install npm for vscode.)
+
+### ssh extension in vscode
+
+Now [ONE](https://github.com/Samsung/ONE) officially releases on linux/ubuntu only. If your develop environment is on Windows, you should use ssh extension in vscode. Let's follow [tutorial](https://code.visualstudio.com/docs/remote/ssh-tutorial).
+
+### workspace
+
+Prepare workspace according to [doc](https://code.visualstudio.com/docs/editor/workspaces). ONE-vscode has been devloped based on workspace.
```

Comment:
```suggestion
Prepare workspace according to the [document](https://code.visualstudio.com/docs/editor/workspaces). ONE-vscode has been developed based on workspace.
```

I'm not sure it's good or bad using abbreviation in official document. (I don't have any preference but just for curiosity)